src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.Shared.xml

```diff
@@ -21,5 +21,8 @@
     <type fullname="System.StartupHookProvider" feature="System.StartupHookProvider.IsSupported" featurevalue="false">
       <method signature="System.Boolean get_IsSupported()" body="stub" value="false" />
     </type>
+    <type fullname="System.Threading.ThreadPool" feature="System.Threading.ThreadPool.EnableDispatchAutoreleasePool" featurevalue="false">
```

Comment:
I think this needs to be the other way. `false` is the default in the code. The feature switch is usually the opposite of the default. See GlobalizationMode.Invariant above.  The default in the code is `false` and setting the feature switch to `true` trims away ICU dependencies.

Which value for this setting trims code? `true` or `false`?

Reply:
Setting this switch to `false` (which is the default in code) should trim out the call to `DispatchItemWithAutoreleasePool`.

Reply:
Ok. Just know that the code won't be trimmed unless someone explicitly sets the switch.

Reply:
I think the feature switch is now set up correctly.

I'm wondering how much code is actually going to be trimmed. It looks like only the `ThreadPoolWorkQueue.DispatchItemWithAutoreleasePool` static method, and 2 extern methods will be able to be trimmed. I still question if this is worth the effort of defining and maintaining a feature switch here.

Reply:
The thread pool's performance is very sensitive to inlining and branching decisions. By enabling trimming here, we can make some of these branches disappear and eek out a little more performance. This one isn't particularly focused on size reduction.